signing-service-challenge/README.md

```diff
@@ -2,23 +2,23 @@
 
 ## Instructions
 
-This challenge is part of the SWE interview process at fiskaly.
+This challenge is part of the software engineering interview process at fiskaly.
 
 If you see this challenge, you've passed the first round of interviews and are now at the second and last stage.
 
-We would ask you to prepare the challenge below to have a basis for discussion in the skill fit interview with two colleagues from the development department.
+We would like you to complete the challenge below. The two colleagues from the development department will then discuss your solution with you in the skill fit interview.
```

Comment:
We don't ask applicants to complete the challenge if they don't find the time. Partial solutions are fine as well.

Reply:
The way it is now phrased sounds like you should know the "two colleagues from the development department"

Reply:
changed to `attempt the challenge`. That fine?